Exercises/NoModules/Chapter 06/Soln6_01.cpp

```diff
@@ -0,0 +1,33 @@
+// Exercise 6-1. Storing odd numbers in an array and accessing them using pointer notation
+
+/* Note that the use of pointer notation is just for the sake of the exercise,           *
+ * to help you understand the intimate relation between pointers and array names.        *
+ * In real code, you'd normally just use array notation, because it is that much easier. */
+
+#include <iostream>
+#include <format>
+
+int main()
+{
+  const size_t n {50};
+  size_t odds[n];
+  for (size_t i {}; i < n; ++i)
+    odds[i] = i + 1;
```

Comment:
How is this "initializes an array with the first 50 odd (as in not even) numbers"?
Your array will contain 1, 2, 3, ...

Reply:
Maybe:
`	odds[0] = 1;
	for (size_t i{1}; i < n; ++i)
		odds[i] = odds[i-1] + 2;
`